data/json/dreams.json

```diff
@@ -206,6 +214,7 @@
     "type": "dream",
     "messages": [
       "You dream of grazing in an open field.",
+      "You dream of a golden calf.",
```

Comment:
It goes straight from dreaming of being a cow to mystic dream about false idoles? I don't think this line fits the tone ^^

Reply:
ok, removing the golden calf ones and references to labrynth